src/nvim/window.c

```diff
@@ -2973,6 +2973,7 @@ int win_new_tabpage(int after)
     return FAIL;
   }
   curtab = newtp;
+  apply_autocmds(EVENT_TABNEW, arg, arg, false, curbuf);
 
   /* Create a new empty window. */
   if (win_alloc_firstwin(tp->tp_curwin) == OK) {
```

Comment:
If I move it into the block, I can move it down to the bottom right way. Should we simply make it like this:

```vim
    apply_autocmds(EVENT_TABNEW, filename, filename, false, curbuf);
    apply_autocmds(EVENT_TABENTER, NULL, NULL, FALSE, curbuf);
    apply_autocmds(EVENT_WINENTER, NULL, NULL, FALSE, curbuf);
```
?